### ISSUE TITLE
WPF: animation freezes (see Teapot sample) if mouse is moving over window

### ISSUE THREAD

**Issue by alariq** [dwmkerr/sharpgl#144]
Not sure if SharpGL related, but this happens. 
Any way to fix this?

**Comment by Omniusz** [comment]
I agree that the issue is the dispatchtimer as Librium said, but if I'm using this library from nuget, and not from this source, I don't have access to change the dispatch timer.

Is it possible instead to use System.Timers.Timer, and then use:              Application.Current.Dispatcher.BeginInvoke( this._dispatcherPriority, new Action(() => { *put rendering action here* }));

Then expose the _dispatcherPriority field via a public property of the usercontrol?

**Comment by DarkDestry** [comment]
In case anyone has this issue still and doesnt want to rebuild the source, you can overwrite the private field using reflection

```
var prop = openGLControl.GetType().GetField("timer",
    System.Reflection.BindingFlags.NonPublic | System.Reflection.BindingFlags.Instance);
prop.SetValue(openGLControl, new DispatcherTimer(DispatcherPriority.Render));
```
